documentation-site/examples/app-nav-bar/basic.js

```diff
@@ -114,11 +118,37 @@ const USER_NAV = [
   },
 ];
 
+function findAndSetActiveChain(item, arr): [boolean, Array<any>] {
```

Comment:
I wonder, can it be somehow part of the nav internals, so folks do not ave to copy-paste every time they use the nav?

Reply:
I wouldn't make it uncontrolled or let the nav component handle what items are currently active, but let the user set it explicitly. This part of the code could potentially be shipped as a helper function but not sure how valuable it would be.

Reply:
I believe there's a helper function for this kind of thing in the tree-view and drag-drop components. An `isItemActive` function may be more useful than storing this information in the array

Reply:
The tree helper function is very specific to the tree component. Additionally, in a real-world application, the active state would probably be defined based on the current url.